// juniper.afi_encap_entry is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap-entry.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_encap_entry;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";
import "juniper/enums/enums.proto";

message AfiEncapEntry {
  message AfiKey {
    ywrapper.StringValue field_data = 53072422;
  }
  message AfiKeyKey {
    juniper.enums.AfiEncapEntryAfiField field_name = 1;
    AfiKey afi_key = 2;
  }
  repeated AfiKeyKey afi_key = 212250188;
  repeated juniper.enums.AfiEncapEntryAfiField dummy = 108175812;
  ywrapper.StringValue name = 145587281;
  ywrapper.StringValue parent_name = 155499100;
  ywrapper.StringValue target_afi_object = 406864802;
}
